Atomic/storage.py

```diff
@@ -37,6 +61,26 @@ def reset(self):
         except:
             selinux.restorecon(root)
 
+    def add(self):
+        dss_conf = "/etc/sysconfig/docker-storage-setup"
+        dss_conf_bak = "/etc/sysconfig/docker-storage-setup" + ".bkp"
+
+        if os.path.exists(dss_conf):
+            shutil.copyfile(dss_conf, dss_conf_bak)
+        elif os.path.exists(dss_conf_bak):
```

Comment:
I think if the dss_conf does not exist,we should throw an exception.  Something is wrong.